src/resources/assets/js/components/enso/bulma/BooleanFilter.vue

```diff
@@ -2,6 +2,7 @@
 
     <vue-filter
```

Comment:
... we need to add :value="value" in the `vue-filter` definition for the cascading `v-model` to work. ... (continued)